packages/twenty-front/src/modules/object-record/record-merge/components/MergePreviewTab.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from '@emotion/styled';
+
+import { CardComponents } from '@/object-record/record-show/components/CardComponents';
+import { SummaryCard } from '@/object-record/record-show/components/SummaryCard';
+import { ObjectRecord } from '@/object-record/types/ObjectRecord';
+import { Section } from 'twenty-ui/layout';
+
+const StyledLoadingContainer = styled.div`
+  align-items: center;
+  color: ${({ theme }) => theme.font.color.secondary};
+  display: flex;
+  justify-content: center;
+  padding: ${({ theme }) => theme.spacing(8)};
+`;
+
```

Comment:
The preview should be read-only, otherwise user can be tempted to modify the merged record directly. When he does that it tries to modify an object that does not exist so it throws an error